lib/tokenize-arg-string.js

```diff
@@ -17,6 +17,7 @@ module.exports = function (argString) {
     // split on spaces unless we're in quotes.
     if (c === ' ' && !opening) {
       if (!(prevC === ' ')) {
+        if (!args[i]) args[i] = ''
```

Comment:
:wave: for this fix, I think it would make sense to move the logic down to the check for the closing quote. So something like:

```
    if (c === opening) {
      if (!args[i]) args[i] = ''
      opening = null
      continue
  }
```

Reply:
I see how that would make sense - placing it at the point of detecting the closing quotes. Both ways pass the tests, so I'll commit the change.

The reason I put it where I did was to ensure that we can't possibly increment `i` before we've checked for a missing argument.

Reply:
Committed :)